code/modules/mob/living/silicon/ai/freelook/chunk.dm

```diff
@@ -114,7 +114,7 @@
 		var/turf/t = turf
 		if(obscuredTurfs[t])
 			if(!t.obscured)
-				t.obscured = image('icons/effects/cameravis.dmi', t, "black", 15)
+				t.obscured = image('icons/effects/cameravis.dmi', t, "black", 16)

```

Comment:
I did a pull request that would standardize all the layers to defines. I'll have to fix that back up; it kept conflicting because of the damn map. For now this quick patch will do.

Besides, doing LIGHTING_LAYER + 1 still means someone would have to edit this actual file to change the relative layer of the static vs the HUD or whatever.